### PR TITLE
Gmcfix

### PR DESCRIPTION
**Pull request by Aftermathrar**
Created at IAmOnYourSide's request.

### CODE REVIEW

**Comment by srliao** [comment]
@HanSeoulOh are you helping with splitting out the itto code?

File: internal/characters/zhongli/abil.go [hunk 1]
```diff
@@ -95,7 +95,7 @@ func (c *char) skillHold(f, max int, createStele bool) {
 	}
 	c.Core.Combat.QueueAttack(ai, core.NewDefCircHit(2, false, core.TargettableEnemy), 0, f-1)
 	//create a stele if none exists and desired by player
```
Comment (HanSeoulOh) [inline internal/characters/zhongli/abil.go:97]:
```suggestion
	//create a stele if less than zhongli's max stele count and desired by player
```

**Review by HanSeoulOh — APPROVED** [review]
lgtm

**Comment by HanSeoulOh** [comment]
@srliao  ready for merge

**Pull request merged**